dishes/peruvian/otras_preparaciones/README.md

```
@@ -0,0 +1,14 @@
+# Otras Preparaciones Peruanas
+
+En esta sección se agrupan preparaciones tradicionales que no encajan en las categorías anteriores, pero son parte fundamental de la cultura culinaria peruana.
+
+## Ejemplos de otras preparaciones
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)